defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 
 # Basic settings
 postgresql_version: 11
+# Short version of the postgresql_version, used in some path and filenames
```

Comment:
Moved `postgresql_version_terse` next to the `postgresql_version`, rather than in a random place down the file.